README.md

```diff
@@ -518,6 +519,46 @@ Since the output of FHIR expression is flexible, users should provide expression
 1. We support FHIR data in R4 and STU 3, JSON format. Support for XML is planned.
 2. De-identification of fields within Extensions is not supported. 
 
+## Using Presidio to de-identify PII in free text fields
+[Presidio](https://github.com/Microsoft/presidio) helps to ensure sensitive data is properly managed and governed. It provides fast identification and anonymization modules for private entities in text such as credit card numbers, names, locations, social security numbers, bitcoin wallets, US phone numbers, financial data and more.
+
+In cases of free text in the FHIR file format, it's possible to anonymize only sensitive data, leaving the rest of the text in place.

```

Comment:
@itye-msft I think it's worth adding the note we have in the Presidio repo about expected de-identification accuracy